crossing-training.lic

```diff
@@ -642,8 +643,13 @@ class CrossingTraining
   end
 
   def train_athletics
-    if DRSkill.getrank('Athletics') >= 400
+    if @climbing_target
       return if @researching
+      walk_to(@climbing_target['id'])
+      climb_practice
+    elsif DRSkill.getrank('Athletics') >= 400
+      return if @researching
+      @climbing_target = get_data('athletics').practice_options['undergondola_branch']
```

Comment:
Since this assigns to `@climbing_target`, does that mean the next time Athletics is trained, `train_athletics` would take the first branch (`climb_practice`) instead of the second (`climb_branch`)?